### PR TITLE
handle cases when AMIs not started in setup

### PR DESCRIPTION
**Pull request by abvgedeika**
after setup of nodes we run 'sudo systemctl status scylla-server.service'
to check scylla status and if 'Active: inactive (dead)' we failed test

### CODE REVIEW

File: sdcm/cluster.py [hunk 1]
```diff
@@ -1356,6 +1356,10 @@ def __init__(self, cluster_uuid=None, cluster_prefix='cluster',
         self.nemesis = []
         self.params = params
         self.add_nodes(n_nodes)
+        for node in self.nodes:
+            status = node.remoter.run('sudo systemctl status scylla-server.service', ignore_status=True)
+            if 'Active: inactive (dead)' in status.stdout:
+                self.fail("some nodes failed to start")
```
Comment (kongove) [inline sdcm/cluster.py]: self.fail() doesn't exist.

We can't assume that nodes has a running scylla-server service after being added to cluster.
We might add a clean/blank node and install and setup scylla in `setup` stage.

This commit blocked all GCE tests, let's revert it?

Comment (roydahan) [inline sdcm/cluster.py]: ok, please revert.

Comment (roydahan) [inline sdcm/cluster.py]: Can we identify the case according to the backend used?

Comment (kongove) [inline sdcm/cluster.py]: @roydahan Yes. If we use AMI, the scylla-server service should be ready(active) after adding node. But for other backends, we might install and setup scylla later, it's normal that the service isn't ready after adding node.
 
I had revert the patch.  @abvgedeika

**Pull request merged**